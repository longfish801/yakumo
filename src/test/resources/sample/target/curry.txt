
【＃タイトル】おいしいカレーの作り方

【－注意：私を信じないで】
　これはサンプルです。
　【ルビ：嘘八百：デタラメ】ばかりですので、ご注意を。
【注意－】

■はじめに

　あなたは【注目：おいしいカレー】の作り方をご存知だろうか。
　生クリーム、チョコレートソース、カスタード、イチゴにバナナにピーチ。この文章を読み終えたとき、あなたは【重要：本当に】おいしいカレーの作り方を知るだろう。【補足：たぶん。】
　たまには【参照：豆のカレー：＃豆カレー】とか食べたくなるよね。

　ところでカレーに必要なものといえば、まずは水【特記：(Water)】すなわちH【下付き：2】Oだろう。もちろん大人【上付き：2】もあるにせよ。

■おいしいカレーはまずサラダから

　あなたはどんなときにカレーを食べたくなりますか。

・お腹が減ったとき。
・過去の栄光【註：栄光】を思いだしたとき。
  ・そう、華麗なる過去を。
  ・あの夏の日、勇気をだしていたら。
  ・きっとハヤシライスを食べていただろう。
・空の青さに感動したとき。

　まあ、人それぞれですよね。
　以下のアンケート回答はいかがでしょうか。

【－変換済：html】
<table class="table">
<tr><th>好きなおかず</th><th>好きなごはん</th></tr>
<tr><td>カレー</td><td>ライス</td></tr>
</table>
【変換済－】

　なんの参考にもなりませんね。
　カレーの辛さには以下の種類があります。

【－用語説明】
【＝用語】甘口
【＝説明】いちばん辛みが穏やか。子供向け。
【＝用語】中辛
【＝説明】中くらいの辛さ。一般的。
【＝用語】大辛
【＝説明】とても辛い。マニアック。
【用語説明－】

□カレーショップに出発進行

　まずは以下のリンクをクリックしてほしい。

=====
*the long fish*
https://longfish801.github.io/
=====

　おわかりいただけただろうか。

【－マスキング】
・：…：・：…：・：…：・：…：・：…：・：…：・：…：・
【マスキング－】

　それでは次のイラスト【註：いらすとや】を三十秒間みつめてほしい。

【＝画像：南蛮そば：img/curry.png】鍋焼きうどん

　おわかりいただけただろうか。

▼ベジタブルカレーは避けろ

　野菜のカレーは極端に甘いことがあります。
　人参ばかり入っている煮物を【傍点：バケツ一杯】食べたことはありますか？ そんな感じです。
　もちろん【訂正：カレーの具は肉さえあればいい】野菜は健康に良いですけどね。

【－コラム：８月３１日はなんの日？】
　８月３１日は野菜の日です。
　野菜を食べましょう。
【コラム－】

▼豆のカレーは通向け＃豆カレー

　カレー屋さんに通い慣れてきたら、豆のカレーにチャレンジしてみましょう。
　豆のカレーといえば、こんな名言を思いだしますね。

=====
　まめに食べろ。豆だけに。
　そんなことを信じていた頃がありました。
【＝引用元】小田牧央『探偵が推理を殺す』
=====

　もっとおいしいカレーのことを知りたければ【リンク：maven：/maven/】を参照し、以下の呪文を唱えましょう。

-----
println 'Hello, Curry World!'
println 'Goodbye, Sukiyaki Night!'
-----

　デザートにはなにか甘いものが良いですね。
　あれ、【参照：はじめに】言いませんでしたっけ？

■謝辞

　おいしいカレーよ、今夜もありがとう。

【＊註記：栄光】
　たとえば自動販売機の足元で五百円玉を拾ったとか。もしくは富士山の登頂に成功した人をテレビ番組で眺めつつポテトチップスを食べたとか。
　ほんの些細なことでもその人にとって良い思い出であれば過去の栄光ですよ。
【註記＊】

【＊註記：いらすとや】
かわいいフリー素材集 いらすとや
https://www.irasutoya.com/
【註記＊】
