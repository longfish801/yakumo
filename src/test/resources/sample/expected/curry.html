
<!doctype html>
<html lang="ja">
<head>
<meta charset="utf-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.0/font/bootstrap-icons.css">
<style type="text/css">
<!--
/* ブロック要素 */
body {
	font-family: "Helvetica Neue", Arial, "Hiragino Kaku Gothic ProN", "Hiragino Sans", Meiryo, sans-serif;
	letter-spacing: 0.1rem;
	line-height: 2.3rem;
	text-align: justify;
	word-wrap: break-word;
	background-color: #f0f0f0;
}
p {
	margin: 1.5rem 0rem;
}
p:only-of-type {
	margin: 0rem 0rem;
}
header#header {
	background-color: #f0f0f0;
}
footer#footer {
	background-color: #f0f0f0;
}
main#main {
	background-color: #ffffff;
}
nav#sidenav {
	background-color: #ffffff;
}
dl#note {
	background-color: #ccffff;
}
div.column {
	padding: 0.5rem;
	margin: 0.5rem 0rem;
	border-width: 1px;
	border-color: #333333;
	border-style: solid;
}
figure > blockquote.blockquote {
	background-color: #ffffcc;
}
figure > figcaption.blockquote-footer {
	background-color: #ffffcc;
}
pre.code {
	background-color: #f0f0f0;
}
/* 目次 */
ul#toc > li.toc-h2 {
	padding-left: 0rem;
}
ul#toc > li.toc-h3 {
	padding-left: 1rem;
}
ul#toc > li.toc-h4 {
	padding-left: 2rem;
}
ul#toc > li.toc-h5 {
	padding-left: 3rem;
}
ul#toc > li.toc-h6 {
	padding-left: 4rem;
}

/* インライン要素 */
span.attention {
	font-size: 1.7rem;
	line-height: 1.7rem;
}
span.dot {
	text-emphasis-style: dot;
	-webkit-text-emphasis-style: filled;
	-moz-text-emphasis-style: filled;
	-ms-text-emphasis-style: filled;
}
-->
</style>
<title>おいしいカレーの作り方</title>
</head>
<body class="container">

<!-- ヘッダー -->
<header id="header" class="row py-3">
<div class="col-lg-8">
<h1>おいしいカレーの作り方</h1>

</div>

</header>

<div class="row justify-content-center">


<nav id="sidenav" class="col-lg-4 py-3">
<!-- 目次 -->
<ul id="toc" class="nav flex-column sticky-top">
<li class="nav-item toc-h2"><a href="#id2_1" class="nav-link">はじめに</a></li>
<li class="nav-item toc-h2"><a href="#id2_2" class="nav-link">おいしいカレーはまずサラダから</a></li>
<li class="nav-item toc-h3"><a href="#id3_3" class="nav-link">カレーショップに出発進行</a></li>
<li class="nav-item toc-h4"><a href="#id4_4" class="nav-link">ベジタブルカレーは避けろ</a></li>
<li class="nav-item toc-h4"><a href="#id4_5" class="nav-link">豆のカレーは通向け</a></li>
<li class="nav-item toc-h2"><a href="#id2_6" class="nav-link">謝辞</a></li>
</ul>
</nav>


<main id="main" class="col-lg-8 py-3">
<!-- 本文 -->
<aside class="alert alert-danger" role="alert">
<header class="alert-heading text-center"><i class="bi bi-exclamation-triangle-fill"></i> 私を信じないで</header>
<p>　これはサンプルです。<br/>
　<ruby>嘘八百<rp>［</rp><rt>デタラメ</rt><rp>］</rp></ruby>ばかりですので、ご注意を。</p>
</aside>

<h2><a name="id2_1"></a>はじめに <div class="float-end fs-6"><a href="#header"><i class="bi bi-chevron-double-up"></i></a></div></h2>

<p>　あなたは<span class="attention">おいしいカレー</span>の作り方をご存知だろうか。<br/>
　生クリーム、チョコレートソース、カスタード、イチゴにバナナにピーチ。この文章を読み終えたとき、あなたは<strong>本当に</strong>おいしいカレーの作り方を知るだろう。<small>たぶん。</small><br/>
　たまには<a href="#id4_5">豆のカレー</a>とか食べたくなるよね。</p>

<p>　ところでカレーに必要なものといえば、まずは水<i>(Water)</i>すなわちH<sub>2</sub>Oだろう。もちろん大人<sup>2</sup>もあるにせよ。</p>

<h2><a name="id2_2"></a>おいしいカレーはまずサラダから <div class="float-end fs-6"><a href="#header"><i class="bi bi-chevron-double-up"></i></a></div></h2>

<p>　あなたはどんなときにカレーを食べたくなりますか。</p>

<ul>
<li>お腹が減ったとき。</li>
<li>過去の栄光<sup><a id="f1" href="#t1">*1</a></sup>を思いだしたとき。
<ul>
<li>そう、華麗なる過去を。</li>
<li>あの夏の日、勇気をだしていたら。<br/>
きっとハヤシライスを食べていただろう。</li>
</ul></li>
<li>空の青さに感動したとき。</li>
</ul>

<p>　まあ、人それぞれですよね。<br/>
　それではカレーを食べる手順はご存知ですか。</p>

<ol>
<li>まずマスクをつけます。</li>
<li>手をよく洗ってください。<br/>
指と指の間や爪の隙間もね。</li>
<li>それでは、いただきます！</li>
</ol>

<p>　以下のアンケート回答はいかがでしょうか。</p>

<table class="table">
<tr><th>好きなおかず</th><th>好きなごはん</th></tr>
<tr><td>カレー</td><td>ライス</td></tr>
</table>

<p>　なんの参考にもなりませんね。<br/>
　カレーの辛さには以下の種類があります。</p>

<dl>
<dt>甘口</dt>
<dd>いちばん辛みが穏やか。子供向け。</dd>
<dt>中辛</dt>
<dd>中くらいの辛さ。一般的。</dd>
<dt>大辛</dt>
<dd>とても辛い。マニアック。</dd>
</dl>

<h3><a name="id3_3"></a>カレーショップに出発進行 <div class="float-end fs-6"><a href="#header"><i class="bi bi-chevron-double-up"></i></a></div></h3>

<p>　まずは以下のリンクをクリックしてほしい。</p>

<figure>
<blockquote class="blockquote p-3">
<p>*the long fish*<br/>
<a href="https://longfish801.github.io/">https://longfish801.github.io/</a></p>
</blockquote>
</figure>

<p>　おわかりいただけただろうか。</p>

<p>・：&hellip;：・：&hellip;：・：&hellip;：・：&hellip;：・：&hellip;：・：&hellip;：・：&hellip;：・</p>

<p>　それでは次のイラスト<sup><a id="f2" href="#t2">*2</a></sup>を三十秒間みつめてほしい。</p>

<figure class="text-center">
<a href="img/curry.png"><img src="img/curry.png" class="img-fluid" alt="南蛮そば"></a>
<figcaption>鍋焼きうどん</figcaption>
</figure>

<p>　おわかりいただけただろうか。</p>

<h4><a name="id4_4"></a>ベジタブルカレーは避けろ <div class="float-end fs-6"><a href="#header"><i class="bi bi-chevron-double-up"></i></a></div></h4>

<p>　野菜のカレーは極端に甘いことがあります。<br/>
　人参ばかり入っている煮物を<span class="dot">バケツ一杯</span>食べたことはありますか？ そんな感じです。<br/>
　もちろん<s>カレーの具は肉さえあればいい</s>野菜は健康に良いですけどね。</p>

<aside class="alert alert-success" role="alert">
<header class="alert-heading text-center"><i class="bi bi-info-circle-fill"></i> ８月３１日はなんの日？</header>
<p>　８月３１日は野菜の日です。<br/>
　野菜を食べましょう。</p>
</aside>

<h4><a name="id4_5"></a>豆のカレーは通向け <div class="float-end fs-6"><a href="#header"><i class="bi bi-chevron-double-up"></i></a></div></h4>

<p>　カレー屋さんに通い慣れてきたら、豆のカレーにチャレンジしてみましょう。<br/>
　豆のカレーといえば、こんな名言を思いだしますね。</p>

<figure>
<blockquote class="blockquote p-3">
<p>　まめに食べろ。豆だけに。<br/>
　そんなことを信じていた頃がありました。</p>
</blockquote>
<figcaption class="blockquote-footer text-end">
<cite>小田牧央『探偵が推理を殺す』</cite>
</figcaption>
</figure>

<p>　もっとおいしいカレーのことを知りたければ<a href="/maven/">maven</a>を参照し、以下の呪文を唱えましょう。</p>

<pre class="code px-3"><code>println 'Hello, Curry World!'
println 'Goodbye, Sukiyaki Night!'</code></pre>

<p>　デザートにはなにか甘いものが良いですね。<br/>
　あれ、<a href="#id2_1">はじめに</a>言いませんでしたっけ？</p>

<div class="column">
<p>　飲むヨーグルトは美味しい。<br/>
　飲まないヨーグルトは味がしない。</p>
</div>

<h2><a name="id2_6"></a>謝辞 <div class="float-end fs-6"><a href="#header"><i class="bi bi-chevron-double-up"></i></a></div></h2>

<p>　おいしいカレーよ、今夜もありがとう。</p>


<!-- 註記 -->
<aside id="appendix">
<dl id="note" class="px-3">
<dt><a name="t1" href="#f1">[*1]</a></dt>
<dd><p>　たとえば自動販売機の足元で五百円玉を拾ったとか。もしくは富士山の登頂に成功した人をテレビ番組で眺めつつポテトチップスを食べたとか。<br/>
　ほんの些細なことでもその人にとって良い思い出であれば過去の栄光ですよ。</p></dd>
<dt><a name="t2" href="#f2">[*2]</a></dt>
<dd><p>かわいいフリー素材集 いらすとや<br/>
<a href="https://www.irasutoya.com/">https://www.irasutoya.com/</a></p></dd>
</dl>
</aside>

</main>

</div><!-- end of row -->

<!-- フッター -->
<footer id="footer" class="row justify-content-center py-3">
<nav id="navi" class="col-lg-12 text-end">
<a href="index.html#tocAll"><i class="bi bi-arrow-up-circle"></i></a>
</nav>
<hr/>
</footer>

<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM" crossorigin="anonymous"></script>

</body>
</html>
